File: CrossSetaDeduplicator/src/CrossSetaWeb/Models/HomeAffairsCitizen.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace CrossSetaWeb.Models
{
    public class HomeAffairsCitizen
    {
        [Required]
        [StringLength(13, MinimumLength = 13)]
        public string NationalID { get; set; }

        [Required]
        [StringLength(100)]
        public string FirstName { get; set; }

        [Required]
        [StringLength(100)]
        public string Surname { get; set; }

        [Required]
        public DateTime DateOfBirth { get; set; }

        public bool IsDeceased { get; set; }
        
        public string VerificationSource { get; set; }
    }
}

File: CrossSetaDeduplicator/src/CrossSetaWeb/CrossSetaLogic/Models/HomeAffairsCitizen.vb
Imports System.ComponentModel.DataAnnotations

Namespace Models
    Public Class HomeAffairsCitizen
        <Required>
        <StringLength(13, MinimumLength:=13)>
        Public Property NationalID As String

        <Required>
        <StringLength(100)>
        Public Property FirstName As String

        <Required>
        <StringLength(100)>
        Public Property Surname As String

        <Required>
        Public Property DateOfBirth As DateTime

        Public Property IsDeceased As Boolean
        
        Public Property VerificationSource As String
    End Class
End Namespace
